packages/cli/test/cli.test.js

```diff
@@ -197,6 +197,32 @@ describe('Lighthouse CI CLI', () => {
       expect(status).toEqual(0);
     }, 60000);
 
+    it('should collect results with a server command with custom start pattern', () => {
```

Comment:
would you mind moving this test to its own test file? maybe `collect-start-server.test.js`?

cli.test.js is by far the longest running bottleneck so would like to avoid increasing its duration if possible :)

Reply:
how about if I move all collect related tests to their own file, something like `ci-collect.test.js`?  I think this way will scale better because later we can move the others to: `ci-upload.test.js`, `ci-sever.test.js`, and so on. 

Or maybe `ci.collect.test.js`

Reply:
let's just move this one out for now, I agree cli.test.js will need to be broken apart eventually but let's leave that refactoring for another day. so far I've just been trying to add all *new* tests into their own file

Reply:
Hey @patrickhulce sorry, I am just watching your comment but I already moved one test to `ci-collect.js`, and added one for the timeout failing case. If you prefer me to change it back, I can do it, no problem. I just went ahead and implemented it after leaving you the first comment. Bad strategy on my part 😅.